dev/io.openliberty.org.jboss.resteasy.common/src/org/jboss/resteasy/specimpl/ResteasyHttpHeaders.java

```diff
@@ -151,13 +153,15 @@ public MediaType getMediaType()
       cachedMediaType = MediaType.valueOf(obj);
       return cachedMediaType;
    }
-
+   
    @Override
    public List<MediaType> getAcceptableMediaTypes()
    {
       List<String> vals = requestHeaders.get(ACCEPT);
-      if (vals == null || vals.isEmpty()) {
-         return Collections.singletonList(MediaType.WILDCARD_TYPE);
+      if (vals == null || vals.isEmpty() 
+                      || (vals.size() == 1 && "*/*".equals(vals.get(0).trim()))) {  // Liberty change
+          return MEDIA_WILDCARD;
```

Comment:
The return of the WILDCARD objects here and the other method should also be marked `// Liberty change`

Reply:
Done